jerry-core/jerry.c

```diff
@@ -140,36 +140,6 @@ jerry_init (jerry_init_flag_t flags) /**< combination of Jerry flags */
   /* Zero out all members. */
   memset (&JERRY_CONTEXT (JERRY_CONTEXT_FIRST_MEMBER), 0, sizeof (jerry_context_t));
 
-  if (flags & (JERRY_INIT_MEM_STATS | JERRY_INIT_MEM_STATS_SEPARATE))
```

Comment:
Yes its true, I set this under the 'mem-stats-separate' option earlier, but it's better to set here.